src/common/utils/serializable-errors.test.js

```diff
@@ -0,0 +1,23 @@
+import serializable from './serializable-errors';
+
+describe('serializable-errors', () => {
+  const error = new Error('Serialized Test Error');
+
+  let representation;
+
+  function roundtrip(value) {
```

Comment:
Interesting approach 🤓

Reply:
Haha 😁 Yeah, I was wondering why I couldn't retrieve the error messages and stack traces in the pop-up window for a bit before it hit me… when they're passed from the content script to the pop-up window, they need to be serializable.

```javascript
JSON.stringify(new Error('Boom!')) === '{}'
```